### DISCUSSION TITLE
Support for Windows 11

### DISCUSSION THREAD

**Discussion by johann8** [rustdesk/rustdesk#820] [General]
Hey,
First, thank you for this beautiful software. At the moment the software does not work under Windows 11. Will the software run under Windows 11 in the near future?
Thanks a lot!

**Comment by 21pages** [comment]
Both lastest release and master are running well on my win11. Could you provide more information?

**Comment by johann8** [comment]
I have found the problem. All the time I entered the FQDN of the server instead of IP address and the Windows 11 could not connect to the server. In logfile it said "os error 10047". The problem is with IPV6. After I entered the IPV4 address instead of FQDN of the server, the problem was solved.
DNS resolution works fine. In firewall I have set up the appropriate rule:

>  rich rules:
        rule family="ipv6" port="21116" protocol="udp" accept 
        rule family="ipv6" port="21115-21119" protocol="tcp" accept

A test with telnet is also successful
> telnet ipv6-address::1 21117

However, Rustdesk has some problems with IPV6.
It was just a coincidence that the other machines did not have IPV6 enabled. That's why I thought it was a problem with Windows 11.

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/issues/152

**Reply by rustdesk** [reply]
and https://twitter.com/rustdesk/status/1538758459373813760
